style.css

```diff
@@ -0,0 +1,295 @@
+@import url("../shared/footer/footer.css");
+@import url("../shared/header/header.css");
+
+:root {
+    --primary-color: #21B6A8;
+    --secondary-color: white;
+    background-color: black;
+    font-family: "Roboto";
+}
+
+html {
+    scroll-behavior: smooth;
+}
+
+body {
+    font-size: 22px;
+    padding: 0;
+    margin: 0;
+    width: 100% !important;
+}
+
+a {
+    text-decoration: none;
+    color: var(--primary-color);
+}
+
+.social-link {
+    color: var(--primary-color);
+    margin-right: 1em;
+    cursor: pointer;
+}
+
+.social-links {
+    display: flex;
+    justify-content: end;
+    right: 0px;
+    float: right;
+}
+
+.panel {
+    display: flex;
+    flex-direction: row;
+

```

Comment:
Suggestion: remove extra spaces between code or add comments to separate them.

Reply:
Sudesh had asked to follow this way for easy understanding of css classes applied to an element.